### DISCUSSION TITLE
an idea of toshiba official RBC-AMT31 schematic

### DISCUSSION THREAD

**Discussion by cambridge2009** [issalig/toshiba_air_cond#21] [Ideas]
https://github.com/burrocargado/toshiba-aircon-mqtt-bridge/blob/master/schematic/KiCad/remote_controller-Toshiba%20Remote%20Controller.png

please take a look at this, here is a toshiba official schematic of remote controller (RBC-AMT31) on Github, from which you can get 5V power form AB line, and TX RX to MCU, might be a goot way to remove external power supply on your board

**Comment by issalig** [comment]
Great! I will take a look at it.
AFAIK, I connected the ESP8266 to some 5V pad on the remote PCB but it drained too much power and the remote could not power on.
Anyway, I will check the scheme, maybe I was connecting it in another place.

Also, in this discussion https://github.com/issalig/toshiba_air_cond/discussions/5#discussioncomment-3387876  there is a circuit by @muxa and also other one borrowed from @bbqkees

**Comment by cambridge2009** [comment]
there is an other picture in same folder which shows how to make 3.3V on H_TAP line from offical schematic. 
Get power from H_TAP should be ok
https://github.com/burrocargado/toshiba-aircon-mqtt-bridge/blob/master/schematic/KiCad/wireless_controller-Wireless%20Controller.png
